src/main/kotlin/no/nav/syfo/melding/database/domain/PPDF.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.syfo.melding.database.domain
+
+import java.time.OffsetDateTime
+import java.util.*
+
+data class PPDF(
```

Comment:
Kanskje `PPdf`? Siden vi bruker `pdf`/`Pdf` andre steder i appen 🤔

Reply:
Jeg gikk for PDF fordi det er en forkortelse, men har ikke veldig sterke følelser knytta til det.